target/cheri-common/cheri_defs.h

```diff
@@ -154,7 +154,16 @@ typedef enum CheriTbFlags {
      * PCC spans the full adddress space and has base zero. This means we do
      * not need to perform bounds checks or subtract/add PCC.base
      */
-    TB_FLAG_PCC_FULL_AS = (1 << 7)
+
```

Comment:
This space here is weird, especially with no comment above the block comment. I'd copy the style for the DDC flags.